README.md

```diff
@@ -1,6 +1,13 @@
 # auth_microservice
 Microservice which abstracts out OAuth2/OpenID exchanges and token management from applications
 
+Requires:
```

Comment:
can you make clear that these are requirements for the example, not for `auth_microservice`. `auth_microservice` doesn't need any of these